split the introduction chapter into introduction, getting started and example (#4)
http-rs/tide-book: src/01-introduction/01-getting_started.md

# Getting started

In order to build a web app in Rust you need an HTTP server, and an async

> runtime. After running `cargo init` add the following lines to your

Reviewer: ```suggestion
runtime. After running `cargo new --bin web-app` add the following lines to your
```

Kind of preference I suppose :shrug:

`Cargo.toml` file:

```toml
# Example, use the version numbers you need

> tide = "0.13.0"

Reviewer: ```suggestion
tide = "0.14.0"
```

> async-std = { version = "1.6.0", features = ["attributes"] }

Reviewer: ```suggestion
async-std = { version = "1.6.5", features = ["attributes"] }
```
